templates/openstackclient/bin/init.sh

```diff
@@ -40,7 +40,7 @@ if [ -d /mnt/ssh-config ]; then
   mkdir -p /home/cloud-admin/.ssh
   cp /mnt/ssh-config/* /home/cloud-admin/.ssh/
   chmod 600 /home/cloud-admin/.ssh/id_rsa
-  chown -R cloud-admin: /home/cloud-admin/.ssh
+  sudo chown -R cloud-admin: /home/cloud-admin/.ssh
```

Comment:
should this always have been sudo? It just happened to work because cloud-admin already owned the contents

Reply:
so far it was only the cloud-admin key, which gets mounted from the secret with correct user iirc.